main.lua

```diff
@@ -115,7 +117,7 @@ return require('./init')(function (...)
   if startRepl == nil and not script then startRepl = true end
 
   if script then
-    require(luvi.path.join(uv.cwd(), script))
+    dofile(luvi.path.join(uv.cwd(), script))
```

Comment:
That's an edge case I don't mind dropping support for.  The closer we can get to vanilla lua behavior the better.